src/components/AdminDashboard/MonitorLogSubmissionStats.jsx

```diff
@@ -0,0 +1,59 @@
+import { React } from 'react';
+import { Text, Box, Flex, Spacer, HStack } from '@chakra-ui/react';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Spacer' is defined but never used.